src/uniparc/components/entry/__tests__/__snapshots__/XRefsSection.spec.tsx.snap

```diff
@@ -89,15 +89,7 @@ exports[`SequenceSection component should render the SequenceSection properly an
                     UniProtKB reviewed
                   </span>
                 </td>
-                <td>
-                  <span>
-                    <a
-                      href="/uniprotkb/P07612/entry"
-                    >
-                      P07612
-                    </a>
-                  </span>
-                </td>
+                <td />
```

Comment:
Something might be off here if it's missing content

Reply:
Content now back.